src/data/polynomial/ring_division.lean

```diff
@@ -185,6 +185,37 @@ begin
   exact root_multiplicity_eq_zero (mt root_X_sub_C.mp (ne.symm hxy))
 end
 
+/-- The multiplicity of `a` as root of `(X - a) ^ n` is `n`. --/
```

Comment:
Docstrings begin with `/--` and end with `-/`:
```suggestion
/-- The multiplicity of `a` as root of `(X - a) ^ n` is `n`. -/
```

Reply:
Ah, sorry for that and thank you.